src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -411,7 +411,8 @@ export const enum GroupChangeKind {
 	EDITOR_CAPABILITIES,
 	EDITOR_PIN,
 	EDITOR_STICKY,
-	EDITOR_DIRTY
+	EDITOR_DIRTY,
+	EDITOR_DISPOSE
 }
 
 export interface IGroupChangeEvent {
```

Comment:
Would it make sense to call this the `IGroupViewChangeEvent` and then keep this separate from a `IGroupModelChangeEvent` so that we can clearly tell apart the one from the other?

Reply:
It might be helpful to have separate interfaces but I still don't know how they would be structurally different

Reply:
I dont mind that both events use the same underlying data structures but then I would suggest the model to define them and the group service to pick them up.

Reply:
Sounds good I'll move it. We want the view to fire the least amount of additional events anyways.